perun-utils/rpc-methods-javadoc-generator/parseRpcMethods.pl

```diff
@@ -633,6 +633,16 @@ sub buildVersion {
 
 		<p>When using JSONP, returned objects are stripped of non relevant properties like <em>createdAt</em>, <em>createdBy</em>, <em>modifiedAt</em>, <em>modifiedBy</em> etc. You can get them when using standard JSON.</p>
 
+		<h3>CSRF Protection</h3>
+
+		<p>CSRF protection is enabled only for state-changing requests (POST, PUT). Also, CSRF is not used in OIDC, since it has its own CSRF protection using own access tokens. Perun will generate a CSRF token during the first API call. The token value is returned to the client in a cookie.The client must read the value from the cookie and return it to the API with each call in the appropriate HTTP header along with the cookie itself. As a result, the token value must be the same in the cookie, HTTP header and server-side session. Domains with both GUI and API deployed are protected against CSRF attacks as a whole. Even only-API clients must implement this protection or use different domain reserved for the API calls, e.g. for:.</p>
```

Comment:
As i said, I would start with the most generic info (about protecting domains), then describe its principle and then specified, that only state-changing requests are protected and protection is not used with OIDC. Basically take this paragraph, split it into three parts and revers the order. I think it will make more sense like this.